tests/network/localnet/conftest.py

```diff
@@ -282,3 +283,84 @@ def nncp_localnet_on_secondary_node_nic(
         worker_node=worker_node1, nodes_available_nics=nodes_available_nics, mtu=mtu
     ) as nncp:
         yield nncp
+
+
+@pytest.fixture(scope="module")
+def vm1_ovs_bridge_localnet_jumbo_frame(
+    namespace_localnet_1: Namespace,
+    ipv4_localnet_address_pool: Generator[str],
+    cudn_localnet_ovs_bridge_jumbo_frame: libcudn.ClusterUserDefinedNetwork,
+) -> Generator[BaseVirtualMachine]:
+    with localnet_vm(
+        namespace=namespace_localnet_1.name,
+        name="localnet-ovs-vm1-jumbo",
+        physical_network_name=cudn_localnet_ovs_bridge_jumbo_frame.name,
+        spec_logical_network=LOCALNET_OVS_BRIDGE_NETWORK,
+        cidr=next(ipv4_localnet_address_pool),
```

Comment:
This fixture is suppose to return a VM with jumbo frame support. That means both the network it is attached to and its own vnic definition are suppose to have MTU defined.
But I do not see the VM created with any special MTU.

What am I missing? How is this passing the testing?